api_sentry.py

```diff
@@ -200,8 +253,50 @@ def issue_insert(self, payload):
             self.db.session.add(issue)
             self.db.session.commit()
 
+    def report_crash_free_rate_payload(self, response_user,
+                                       response_session, release_version):
+        # Crash free rate is a float. Convert it to percentage.
+        session_rate = response_session['groups'][0]['totals'].get(
+            'crash_free_rate(session)', None)
+        user_rate = response_user['groups'][0]['totals'].get(
+            'crash_free_rate(user)', None)
+        # Sometimes the REST API calls return null values in the field
+        # Return None if either rate is null
+        if session_rate is not None and user_rate is not None:
+            percentage_session_rate = round(session_rate * 100, 3)
+            percentage_user_rate = round(user_rate * 100, 3)
+        else:
+            return None
+        now = DatetimeUtils.start_date('0')
```

Comment:
Setting `num_days` to `0` makes the `DatetimeUtils.start_date` to return an empty string. To trigger the function to return `datetime.now()`, I set `num_days` to `'0'` instead.
https://github.com/mozilla-mobile/testops-dashboard/blob/master/utils/datetime_utils.py#L32